.envrc

```diff
@@ -0,0 +1,2 @@
+GIT_ROOT=$(git rev-parse --show-toplevel)
```

Comment:
I'm not 100% sure if we should include this file in our repo. This would definitely mess with my personal setup since I have some things configured in a `.envrc` file of my own. 

I can't really find any best practices on the direnv github, but I feel the configuration of an environment is something very user-specific and prone to personal preference.

Reply:
I agree with you, the `.envrc` should be used for the personal github repository,  i moved to taskfile vars section